iptables/table.go

```diff
@@ -214,6 +214,12 @@ type Table struct {
 	// to top-level chains.
 	insertMode string
 
+	// Record when we did our most recent updates and refreshes of the table.  We use these to
+	// calculate the next time we should force a refresh.
+	lastUpdateTime  time.Time
+	lastRefreshTime time.Time
```

Comment:
(Or perhaps just 'refresh' to 'read' or 'check', if you don't want to churn pre-existing uses of 'update'.)